packages/launch/src/focused-launch/summary/style.scss

```diff
@@ -119,20 +132,69 @@
 			margin-right: 12px;
 		}
 	}
+
+	@include break-large {
+		width: $commentary-column-width-large;
+	}
 }
 
 .focused-launch-summary__data-input {
-	flex: 1;
+	flex-grow: 1;
+	flex-shrink: 1;
+}
+
+.focused-launch-summary__details-link {
+	display: block;
+	margin-top: 14px;
+	color: var( --studio-gray-60 );
+}
+
+.focused-launch-summary__actions-wrapper {
+	@include break-small {
+		display: flex;
+		justify-content: space-between;
+		align-items: center;
+	}
 
 	@include break-medium {
-		flex: 0.6;
+		flex-direction: column-reverse;
+		align-items: flex-start;
+		gap: 20px;
+		width: calc( 100% - #{$commentary-column-width-medium + $commentary-column-gap-small} );
+	}
+
+	@include break-large {
+		flex-direction: row;
+		align-items: center;
+		width: calc( 100% - #{$commentary-column-width-large + $commentary-column-gap-large} );
+	}
+
+	@include break-wide {
+		width: calc( 100% - #{$commentary-column-width-large + $commentary-column-gap-wide} );
 	}
 }
 
-.onboarding-title {
-	@include onboarding-font-recoleta;
+.focused-launch-summary__ask-for-help {
+	// to make scrolling space for sticky action bar
+	margin-bottom: 80px;
```

Comment:
Not a blocker but I would suggest using the `$onboarding-footer-height` variable.

Reply:
Updated! Thank you for the suggestion